src/multibody/geometry.hpp

```diff
@@ -220,20 +232,35 @@ namespace se3
     std::vector <DistanceResult> distance_results;
     
     ///
+    /// \brief Defines what information should be computed by collision test.
+    ///
+    fcl::CollisionRequest collisionRequest;
```

Comment:
I do not see any use for this but, yes, it could be. Do you see any use for it ?